src/OpenTelemetry/Metrics/MeterProviderBuilderExtensions.cs

```diff
@@ -107,7 +108,20 @@ public static MeterProviderBuilder AddView(this MeterProviderBuilder meterProvid
             if (metricStreamConfiguration is ExplicitBucketHistogramConfiguration histogramConfiguration)
             {
                 // Validate histogram boundaries
-                if (histogramConfiguration.Boundaries != null && !IsSortedAndDistinct(histogramConfiguration.Boundaries))
+                if (histogramConfiguration.BucketBounds == null ||
```

Comment:
The user can provide `Boundaries` as `null`. In this case, we use the default bucket distribution for the Histogram: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#explicit-bucket-histogram-aggregation

Reply:
Yes, I am doing that here: https://github.com/open-telemetry/opentelemetry-dotnet/pull/2735/files#diff-a149c9e64379e14c17737caf055686e54be18c08fb57d4f129576a141f61f740R96